src/Angor/Client/Pages/Investor.razor

```diff
@@ -402,7 +400,7 @@
         var NostrDMPrivateKeyHex = Encoders.Hex.EncodeData(NostrDMPrivateKey.ToBytes());
         var NostrDMPubkey = _DerivationOperations.DeriveNostrPubKey(words, 1);
 
-        await checkSignatureFromFounder();
+        CheckSignatureFromFounder();
```

Comment:
no `await`, on purpose?

Reply:
The logic in that method is triggered by the end of stream event from the relay